stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -22,6 +25,9 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+{{- if .Values.controller.rbac.enabled }}
+      serviceAccountName: {{ template "fullname" . }}
+{{- end }}
```

Comment:
I submitted a similar update for Traefik in https://github.com/kubernetes/charts/pull/1225. When switching RBAC enabled from true to false and helm installing, I found the serviceAccountName stays as the Traefik specific one rather than being reset to `default`. I wondered if you had encountered that issue here?